credential/exchange/submission_test.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func TestBuildPresentationSubmission(t *testing.T) {
 	t.Run("Unsupported embed target", func(tt *testing.T) {
-		_, err := BuildPresentationSubmission(crypto.JWTSigner{}, PresentationDefinition{}, nil, "badEmbedTarget")
+		_, err := BuildPresentationSubmission(crypto.JWTSigner{}, "requester", PresentationDefinition{}, nil, "badEmbedTarget")
```

Comment:
if "requester" and "submitter" are things we will use mostly in production may be good to make a const

Reply:
not used in production as these strings. I would imagine they're each the DIDs of the requester/submitter